client/src/containers/App.tsx

```diff
@@ -75,6 +237,7 @@ export default class App extends Component<Props, State> {
                   </span>
                 )}
                 <nav className="inline">
+                  {featureCalloutWidget}
```

Comment:
So you might want to consider actually moving all the feature callout widget logic out of `App.tsx` and into a separate component in a different file, e.g. `FeatureCalloutWidget.tsx`.  Then this line would become `<FeatureCalloutWidget />` instead of `{featureCalloutWidget}` and it'd be a lot easier to understand the widget's logic because it wouldn't be intermingled in all the general WoW app logic.  It would also make the feature callout widget a lot easier to reuse in our other products!

Reply:
I did this in a recent commit!